docs/ui/storage/s3-image-picker.md

```diff
@@ -0,0 +1,12 @@
+---
+title: S3 Image Picker
+description: S3 Image Picker UI Component
```

Comment:
Let's improve the md `description` for each of these components. This is what shows up in Google and other search engines. Let's briefly describe what this component allows a user to do

Reply:
I can add it in.